### PR TITLE
remove default config

### LINKED ISSUE #2654: Remove default config from simorgh
**Is your feature request related to a problem? Please describe.**
Simorgh has default config which isnt actually used by anything, we should give it the 🥾

**Describe the solution you'd like**
Remove default config from simorgh

Delete `defaultConfig` from https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/index.js#L7

Delete the file https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/default.js

On https://github.com/bbc/simorgh/blob/latest/src/app/contexts/ServiceContext/index.jsx#L8 just replace `React.createContext(defaultService)` with `React.createContext({})`

There may/will be other places in the tests that rely on/use the default config. We should stub/fix these cases to either use fake data or real data, like using service `news` to ensure it uses real data.

This _should_ have zero impact on the main application, as 'default' values are never used.

**Testing notes**
[Tester to complete]

Dev insight: No new tests needed, just ensure current pass

**Additional context**

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #2654 

**Overall change:** 

Removes the default config, which is unused in the main application and mostly touches a few tests. Updates the affected tests.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by thekp — CHANGES_REQUESTED** [review]
we could possibly fix the code climate issue: https://codeclimate.com/github/bbc/simorgh/pull/3407

by creating a function with the repetitive contextProvider logic. 
i.e: something similar to this: https://github.com/bbc/simorgh/blob/latest/src/app/containers/Paragraph/index.test.jsx#L66-L70

**Review by thekp — APPROVED** [review]
image

looks good to me 👍

File: src/app/containers/ArticleTimestamp/index.test.jsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ describe('ArticleTimestamp', () => {
   describe('with no data', () => {
     suppressPropWarnings(['firstPublished', 'undefined']);
     suppressPropWarnings(['lastPublished', 'undefined']);
-    isNull('should return null', <ArticleTimestamp />);
+    isNull('should return null', <WrappedArticleTimestamp />);
```
Comment (thekp) [inline src/app/containers/ArticleTimestamp/index.test.jsx]: i'm not too sure why this has changed 🤔

**Review by dr3 — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**